src/core/server/models/comment/revision.ts

```diff
@@ -35,6 +37,36 @@ export interface RevisionMetadata {
   nudge?: boolean;
 }
 
+export interface CommentEmbedMedia {
+  original: string;
+  still?: string;
+  video?: string;
+}
+
+/**
+ * CommentEmbed stores the source and url of any links
+ * that were detected as embeddable content in the comment body.
+ */
+export interface CommentEmbed {
+  /**
+   * url represents the actual linked content for the embed.
+   */
+  url: string;
+
+  /**
+   * type represents which source the link comes from.
+   */
+  source: GQLEMBED_SOURCE;
+
+  title?: string;
+
+  remote_id?: string;
+
+  media?: CommentEmbedMedia;
+  height?: string;
+  width?: string;
+}
```

Comment:
This looks pretty identical to the Graph generated interface, prefer that over this one.